spring-boot-modules/spring-boot-properties-migrator-demo/src/main/java/com/baeldung/spring/boot/properties/migrator/Main.java

```diff
@@ -0,0 +1,11 @@
+package com.baeldung.spring.boot.properties.migrator;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class Main {
+	public static void main(String[] args) {
+		SpringApplication.run(Main.class, args);
+	}
```

Comment:
Let's use spaces instead of tabs for the indentation.

Reply:
Ahh sure. I had changed the formatter in between fixed now.